README.md

```diff
@@ -2,7 +2,7 @@
 <h1 align="center">Haste</h1>
 <p align="center">An extendable, blazing fast build system that cares about user experience</p>
 
-<p align="center"><strong>Haste is still in an early development stage and it is lacking in features and documentation. Please try it out, give feedback, and help fix bugs.</strong></p>
+<p align="center"><strong>Haste is still in an early development stage and is lacking in features and documentation. Please try it out, give feedback, and help fix bugs.</strong></p>
```

Comment:
I think it's clearer with the `it`, so I wouldn't change it.

Reply:
Yeah, I kinda felt I was nitpicking when I changed it anyways. It really is fine either way.